reportmanager/build/assets/angular/angular-resource.unminified.js

```diff
@@ -0,0 +1,695 @@
+/**
```

Comment:
Even entire _build_ folder does not required. It will generate once build is run and should be specific to local.